index.html

```diff
@@ -70,12 +70,17 @@ <h3 id="advantages">Advantages of wheels</h3>
                 <h2 id="about-list">What is this list?</h2>
                 <p>This site shows the top 360 most-downloaded packages on <a href="https://pypi.org/">PyPI</a> showing which have been uploaded as wheel archives.</p>
                 <ul>
-                    <li><span class="text-success">Green</span> packages offer wheels,</li>
+                    <li><span class="text-success">Green</span> packages offer wheels compatible with the newest Python version (updated shortly before its final release),</li>
+                    <li><span class="text-warning">Orange</span> packages offer older wheels,</li>
                     <li><span class="text-muted">White</span> packages have no wheel archives uploaded (yet!).</li>
                 </ul>
+                <p>Currently, Python 3.13 is considered the newest Python version.</p>
                 <p>Packages that are known to be deprecated are not included. (For example distribute). If your package is incorrectly listed, please <a href="https://github.com/meshy/pythonwheels/issues/">create a ticket</a>.</p>
                 <p>This used to show the all-time most-downloaded packages. The all-time list is no longer available, and the packages in <a href="https://hugovk.github.io/top-pypi-packages/">the last-30-days list</a> will change to reflect more closely what the Python community is using.
                 <p>This is not the official website for wheels, just a nice visual way to measure adoption. To see the authoritative guide on wheels and other aspects of Python packaging, see the <a href="https://packaging.python.org">Python Packaging User Guide</a>.</p>
+                <h2 id="creating-new-version-wheels">My package is orange. What can I do?</h2>
+                    <p>If you have a package that doesn't publish the wheels for the newest Python yet, consider releasing them on PyPI. This will make it possible for Python users to start developing with a new Python version from the day of its release.</p>
```

Comment:
Can we move this section to after the "My package is white" section, please?

I'd like to make some changes to the wording here:

```suggestion
                    <p>This indicates that your package doesn't yet publish the wheels compatible with the latest Python. Consider releasing a new compatible build or version, to make it possible for users of your package to update Python.</p>
```

For the following paragraph: I don't know much about the stable ABI, and I think it's worth going into a little more detail about the impact that porting would make. Would it mean that it's possible to release a universal wheel?